runtime/permissions/lib.rs

```diff
@@ -2187,6 +2175,14 @@ impl AllowRunDescriptor {
       AllowRunDescriptor(path),
     ))
   }
+
+  pub fn is_path(text: &str) -> bool {
+    if cfg!(windows) {
+      text.contains('/') || text.contains('\\') || Path::new(text).is_absolute()
+    } else {
+      text.contains('/')
+    }
+  }
```

Comment:
This is used in the permissions in the config file PR and I'm too lazy to remove it from this one at this point (copy and pasted some changes from the other pr).